assets/python/course-downloader/run.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# Specify the name of the Python script
+python_script="obtain_course_list.py"
+
+# Unused
+exit 1
+
+# Check if the Python script exists
+if [ -f "$python_script" ]; then
```

Comment:
Instead of explicitly checking the existence of the script you can use `set -e` in the second line to have the entire script fail when a subcommand fails

Reply:
Would there be any improvement with this change as there currently isnt any way of handling these errors.
Also this if statement should never go into the else branch. Going into else should signal a misconfiguration of the dockerfile (Relevant COPYs missing) or the run.sh script itself (python_script variable wrong)

Reply:
Yep, `set -e` fails the entire script if the python script fails, leading to easier debugging in case of errors

Reply:
Resolved in e39c172f23cea37d84a26ec4d008708f5349c1ba